### PR TITLE
Url shortener added

### PR DESCRIPTION
**Pull request by cagataycali**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[url-shortener](https://github.com/cagataycali/url-shortener) -Shitty url shortener, emoji and ai powered. ([Demo](http://🌍✌🏼.ws), [Source Code](https://github.com/cagataycali/url-shortenere)) `MIT` `JavaScript```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
url-shortener |   * [url-shortener](https://github.com/cagataycali/url-shortener) - Shitty url shortener, emoji and ai powered. 🌍✌🏼 `MIT` `JavaScript` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by nodiscc** [comment]
@cagataycali thanks for the pull request, there are a few problems with this entry


 * Unicode characters/emojis are not useful in the description and our syntax checks do not play nice with them, they should be removed;
 * The project has _zero_ deployment/installation/configuration instructions. What are the server side requirements? How do you install it?

We don't enforce specific quality requirements but listed projects should at least have some basic documentation/install instructions. Knowing the server side requirements is also important for the language/platform tag (I can see it's written in JS, does it require Nodejs? Is it only compatible with heroku? etc.) Thanks in advance

**Comment by cagataycali** [comment]
Thank you, I will arrange and push them back.

**Comment by cagataycali** [comment]
Hello again,
I fixed problems.
Thanks
Cagatay

**Comment by cagataycali** [comment]
You are awesome, thank you. 🥇

**Comment by cagataycali** [comment]
I'll add the warning badge, I used a third party service for long tail urls emoji prediction.

File: README.md [hunk 1]
```diff
@@ -1130,6 +1130,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Nimbus](https://github.com/ethanal/nimbus) - A URL shortener and file sharer with a drag-and-drop OS X menu bar client and web interface. `MIT` `Python`
   * [Polr](https://project.polr.me/) - A modern, minimalist, modular, and lightweight URL shortener. ([Source Code](https://github.com/Cydrobolt/polr)) `GPLv2` `PHP`
   * [shuri](https://github.com/pips-/shuri) - SHort URI - Lighweight URL shortener. `MIT` `PHP`
+  * [url-shortener](https://github.com/cagataycali/url-shortener) - `⚠`  Shitty url shortener, emoji and ai powered. `MIT` `JavaScript`
```
Comment (nodiscc) [inline README.md]: ⚠ must go right next to the title. Look at other entries.

**Comment by cagataycali** [comment]
Thanks a lot 👍

**Pull request merged**